website/content/docs/connect/cluster-peering/usage/create-cluster-peering.mdx

```diff
@@ -0,0 +1,834 @@
+---
+layout: docs
+page_title: Establish Cluster Peering Connections
+description: >-
+  Generate a peering token to establish communication, export services, and authorize requests for cluster peering connections. Learn how to create, list, read, check, and delete peering connections.
+---
+
+# Establish cluster peering connections
+
+This page details the process for establishing a cluster peering connection between services deployed to different datacenters. You can interact with Consul's cluster peering features using the HTTP API, the CLI, the UI, or the Kubernetes controller. While the specific steps vary slightly for each, the overall process for establishing a cluster peering connection consists of the following steps:
+
+1. Create a peering token in one cluster.
+1. Use the peering token to establish peering with a second cluster.
+1. Export services between clusters.
+1. Create intentions to authorize services for peers.
+
+Cluster peering between services cannot be established until all four steps are complete.
+
+For more information about using Kubernetes with cluster peering, including required Helm values, required CRDs, and other configuration options, refer to [cluster peering on Kubernetes](/consul/docs/connect/cluster-peering/usage/k8s)
+
+## Requirements
+
+To establish a cluster peering connection, your deployments must meet several requirements. For additional guidance on required configuration, refer to the [cluster peering technical specifications](/consul/docs/connect/cluster-peering/configuration).
+
+<Tabs>
+<Tab heading="HTTP API" group="api">
+
+### VMs and on-prem deployment requirements
+
+You must meet the following requirements to use cluster peering:
+
+- Consul v1.13 or higher
+- Services hosted in admin partitions on separate datacenters
+
+If you need to make services available to an admin partition in the same datacenter, do not use cluster peering. Instead, use the [`exported-services` configuration entry](/consul/docs/connect/config-entries/exported-services) to make service upstreams available to other admin partitions in a single datacenter.
+
+### Mesh gateway requirements
+
+Mesh gateways are required for all cluster peering connections. Consider the following architectural requirements when creating mesh gateways:
+
+- A registered mesh gateway is required in order to export services to peers.
+- For Enterprise, the mesh gateway that exports services must be registered in the same partition as the exported services and their `exported-services` configuration entry.
+- To use the `local` mesh gateway mode, you must register a mesh gateway in the importing cluster.
+
```

Comment:
are these mesh gateway requirements the same? if so, we should use partials so we don't repeat stuff (and make maintenance easier for us)